qbraid/transpiler/conversions/cirq/cirq_to_qasm2.py

```
@@ -31,6 +32,44 @@
+    if len(positions) < 2:
+        return qasm
+
+    def sort_key(index: int) -> tuple:
+        name = _CREG.match(lines[index])["name"]
+        match = _BIT_INDEX.match(name)
+        if match is None:
+            return (1, "", 0, index)
+        return (0, match["register"], int(match["index"]), index)
+
+    reordered = [lines[i] for i in sorted(positions, key=sort_key)]
+    for slot, line in zip(positions, reordered):
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)